### PR TITLE
Change file permissions on console serving cert

### PR DESCRIPTION
**Pull request by spadgett**
This avoids the console crash looping with system:authenticated is added to scc/anyuid.

/assign @sdodson

### CODE REVIEW

**Comment by spadgett** [comment]
This is why it wasn't working when running with `anyuid`:

```
sh-4.2$ id
uid=1001 gid=0(root) groups=0(root)
```

**Comment by php-coder** [comment]
If a pod was running under 1001 UID, why the secrets are owned by `root`? And why it worked when a pod was running with `restricted` SCC and had UID like 10005000?

**Comment by spadgett** [comment]
> If a pod was running under 1001 UID, why the secrets are owned by root?

@sjenning Is this expected?

> And why it worked before when a pod was running with restricted SCC and had UID like 10005000?

I think because we were using the decimal `400` which is not the same as octal `0400`. The files were assigned these permissions previously:

```
-rw-rw---- 1 root 1000060000 2469 May 29 14:13 tls.crt
-rw-rw---- 1 root 1000060000 1679 May 29 14:13 tls.key
```

**Comment by spadgett** [comment]
Oddly when they get different permissions after I run the command `oc adm policy add-scc-to-group anyuid system:authenticated`:

```
-rw--w---- 1 root root 2469 May 29 14:17 tls.crt
-rw--w---- 1 root root 1679 May 29 14:17 tls.key
```

**Comment by php-coder** [comment]
I'm also surprised by the fact that group has been changed (1000060000 -> root) while I'd expect that it should always be the same (root).

File: roles/openshift_web_console/files/console-template.yaml [hunk 1]
```diff
@@ -87,11 +87,11 @@ objects:
         volumes:
         - name: serving-cert
           secret:
-            defaultMode: 400
+            defaultMode: 0040
```
Comment (sdodson) [inline roles/openshift_web_console/files/console-template.yaml]: Why not 0440 here too?

**Comment by sdodson** [comment]
I'm not sure I'm familiar enough with SCCs or how secret mounts are handled.

**Comment by sdodson** [comment]
bot, retest this please

**Pull request merged**